base-template.sh

```diff
@@ -67,6 +67,8 @@ set_main_variables() {
         CURRENT_GIT_DIR=".git" # reset to a sensible default
     fi
 
+    INSTALL_DIR=$(git config --global githooks.installDir)
```

Comment:
I'd check with every usage if `[ -z "$INSTALL_DIR" ]` and set it to `$HOME` if it is. Might be worth extracting it into a function.

I wonder why tests that don't install first didn't fail. When this config is missing, the path would become relative to root. Sounds like I have some missing coverage here. :(

Reply:
Oh if this is the common variable setup function then we don't need to extract setting this up further, just an extra if here. Please do the same in the other files too.

Reply:
I will check during `set_main_variables`

Reply:
> Sounds like I have some missing coverage here. :(
Hm... strange